lib/screens/settings/about.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:expandable/expandable.dart';
+
+class AboutPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: Text("About")),
+      body: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.all(2.0),
+          child: Column(
+            children: <Widget>[
+              Card(
+                semanticContainer: true,
+                clipBehavior: Clip.antiAliasWithSaveLayer,
```

Comment:
Is it necessary? `Clip.antiAliasWithSaveLayer` is much slower than  `Clip.antiAlias`. But idk, maybe it's necessary in this situation, I didn't pull it and check on my machine.

Reply:
Just did it for a smoother edge (not really visible). We can do ```hard edge``` for max speed btw :). Will change to AntiAlias and push asap